jest-mocks/globalMocks.js

```diff
@@ -0,0 +1,2 @@
+global.__STATIC_PATH__ = "https://uebergrape.staging.chatgrape.com/"
+global.__DEV__ = true
```

Comment:
we don't need `__TEST__` anymore?

Reply:
It's never used so far. I only wanted to add things that we really need for now.